pkg/api/org_test.go

```diff
@@ -116,13 +119,14 @@ func TestAPIEndpoint_PutCurrentOrg_LegacyAccessControl(t *testing.T) {
 }
 
 func TestAPIEndpoint_PutCurrentOrg_AccessControl(t *testing.T) {
+	var err error
 	sc := setupHTTPServer(t, true)
 	setInitCtxSignedInViewer(sc.initCtx)
 
-	err := sc.db.CreateOrg(context.Background(), &models.CreateOrgCommand{Name: "TestOrg", UserId: sc.initCtx.UserID})
+	sc.hs.orgService, err = orgimpl.ProvideService(sc.db, sc.cfg, quotatest.New(false, nil))
```

Comment:
nit; for uniformity with above we can use `sc.hs.QuotaService` (which points to the [fake](https://github.com/grafana/grafana/blob/9bdfd1729c33e39b77135e38e4ec701a638d3438/pkg/api/common_test.go#L376) service)